api/organisations/invites/views.py

```diff
@@ -44,6 +46,15 @@ def join_organisation_from_email(request, hash):
         error_data = {"detail": str(e)}
         return Response(data=error_data, status=status.HTTP_400_BAD_REQUEST)
 
+    hubspot_cookie = request.COOKIES.get(HUBSPOT_COOKIE_NAME)
+
+    if hubspot_cookie:
+        if not HubspotTracker.objects.filter(user=request.user).exists():
+            HubspotTracker.objects.create(
```

Comment:
Do we need to update the stored cookie in case it's regenerated for some reason?

Reply:
Sure, I've switched the code to an `update_or_create` method to handle that if it ever comes up.